packages/flutter_web_plugins/lib/src/navigation/url_strategy.dart

```diff
@@ -15,7 +13,7 @@ import 'utils.dart';
 ///
 /// Setting this to null disables all integration with the browser history.
 void setUrlStrategy(UrlStrategy strategy) {
```

Comment:
According to the dartdoc this function allows null argument.

Reply:
Yes, null is a valid value here to disable url strategies altogether. The signature should be:
```dart
void setUrlStrategy(UrlStrategy? strategy) {
```

Reply:
Ah, makes sense, this also requires the js-interop typedef to be:

```
typedef _JsSetUrlStrategy = void Function(JsUrlStrategy?);
```

So I can pass 'null' to the JS.